src/inject/dynamic-theme/index.ts

```diff
@@ -49,7 +49,8 @@ function stopStylePositionWatchers() {
 }
 
 function createStaticStyleOverrides() {
-    const fallbackStyle = createOrUpdateStyle('darkreader--fallback');
+    const fallbackStyle = createOrUpdateStyle('darkreader--fallback', document);
+    createOrUpdateStyle('darkreader--fallback', document).remove();
```

Comment:
Are you trying to remove it for placing into HEAD later? No need, `insertBefore` will do this.

Reply:
Got ya